### DISCUSSION TITLE
Reduce dirt textures on instruments

### DISCUSSION THREAD

**Discussion by Phil787-9** [magknight/787aviatorsTracker#528] [Completed things] [answered]
The current Beta Version introduced "dirty" instrument textures, which I don't quite fancy. Can we please revert to the textures from 1.6.6 stable version?

**Comment by aeroniemi** [comment, marked as answer]
The dirt was reduced in v1.7.4

**Comment by KBNOtaku** [comment]
Some of the new liveries have reduced dirt textures included!
https://magknight.org/liveries/
